### DISCUSSION TITLE
NoDriver response interceptor does not intercept all response.

### DISCUSSION THREAD

**Discussion by EricYoong** [ultrafunkamsterdam/undetected-chromedriver#1930] [General]
Hi @ultrafunkamsterdam/all,

I have a question regarding the Nodriver response interceptor which it seems like does not intercept all response as it should
E.G: From the Chrome Dev Tools we can see that it loaded **https://www.google.com.my/pagead/1p-conversion/853901910** However in the response interceptor it does not print it out.
![image](https://github.com/ultrafunkamsterdam/undetected-chromedriver/assets/28799051/56ed536f-1ca4-4cec-b54a-000499f9cc7b)

This is how I did to intercept all the response using nodriver
```
async def listen(self, page):
        async def handler(evt:nodriver.cdp.network.ResponseReceived):
            async with self.lock:
                    self._logging.log(f"RESPONSE EVENT PERCEIVED BY BROWSER IS:- {evt.type_} {evt.response.url}") # If unsure about event or to check behaviour of browser
        page.add_handler(nodriver.cdp.network.ResponseReceived, handler)
```
And this is how i called it
```
await self.browser.main_tab.send(nodriver.cdp.network.enable(max_post_data_size=65536))
await self.browser.main_tab.send(nodriver.cdp.network.set_attach_debug_stack(True))
await self.listen(self.browser.main_tab)
```
Can you help me out this issue? Tried to figure it out for passed few days but I still not able to do so.

**Comment by EricYoong** [comment]
Also the request intercept does not intercept this request.

**Reply by epicwhale** [reply]
Were you able to find a fix for this? I am facing a slightly similar bug when listening to the fetch.RequestPaused event, where I don't get any event except for the first network request... https://github.com/ultrafunkamsterdam/undetected-chromedriver/discussions/1972

**Comment by EricYoong** [comment]
also it seems like if the request was made in the injected javascript it seems to be not able to capture the request due to the different Target is different due to the script is being injected when dom is ready. I tried to set auto attach to true but yet it is still not working
```
await self.browser.main_tab.send(nodriver.cdp.target.set_auto_attach(True, True))
```
